x/examples/fyne-proxy/main.go

```diff
@@ -126,6 +127,7 @@ func makeAppHeader(title string) *fyne.Container {
 }
 
 func main() {
+	defer sysproxy.DisableWebProxy()
```

Comment:
Disable on runningProxy.Close() instead.

Perhaps have a defer that checks if proxy != nil and closes it.

Move it to where the proxy variable is defined.